day-1.md

```
@@ -1,4 +1,16 @@
-# Daily Learning
+  # Daily Learning
```

markdownlint-cli2 (0.17.2)

Headings must start at the beginning of the line

(MD023, heading-start-left)